core/capabilities/triggers/logevent/logeventcap/event_trigger-schema.json

```diff
@@ -0,0 +1,75 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/smartcontractkit/chainlink/core/capabilities/triggers/logevent/log-event-trigger",
+    "$defs": {
+        "head": {
+            "type": "object",
+            "properties": {
+                "Height": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "Hash": {
+                    "type": "string",
+                    "minLength": 42
+                },
+                "Timestamp": {
+                    "type": "integer",
+                    "minimum": 10000
```

Comment:
strange minimum to choose.  I doubt anything goes that far back in time.  I don't know if there's a meaningful minimum to include

Reply:
I was thinking of setting minimum time at some arbitrary timestamp like Jan 1, 2020 midnight. But then thought, some simulated backend tests have time set to 0 or 86400 when they start. Not sure, what would be a good minimum to set

Reply:
I probably just wouldn't set one then, otherwise, 0 seems reasonable.

Reply:
Sure